docs/src/reference/asciidoc/appendix/release-notes/8.0.0.adoc

```diff
@@ -0,0 +1,58 @@
+[[eshadoop-8.0.0]]
+== Elasticsearch for Apache Hadoop version 8.0.0
+
+The following list are changes in 8.0.0 as compared to 7.17.0, and combines
+release notes from the 8.0.0-alpha1, -alpha2, -beta1, -rc1 and -rc2 releases.
+
+[[breaking-8.0.0]]
+[float]
+=== Breaking Changes
+
+- Deprecate support for Spark 1.x, Scala 2.10 on Spark 2.x, Hadoop 1, Pig, and Storm
+https://github.com/elastic/elasticsearch-hadoop/pull/1845[#1845]
+
+Core::
+- Boost default scroll size to 1000
+https://github.com/elastic/elasticsearch-hadoop/pull/1429[#1429]
+
+[[new-8.0.0]]
+[float]
+=== Enhancements
+
+Build::
+- Upgrade build to use JDK 12 and current build tools
+https://github.com/elastic/elasticsearch-hadoop/pull/1351[#1351]
+
+- Build Hadoop with Java 17
+https://github.com/elastic/elasticsearch-hadoop/pull/1808[#1808]
+
+Build/Gradle::
+- Upgrade to and fix compatibility with Gradle 5.5
```

Comment:
This was in 7.6.1.

Reply:
Removed in https://github.com/elastic/elasticsearch-hadoop/pull/1902/commits/8237c6aa68af6eba861502d4b4cca5b532300e30